docker/README.MD

```diff
@@ -0,0 +1,23 @@
+# Docker & Nvidia-docker install
+1. First install [docker-ce](https://docs.docker.com/install/linux/docker-ce/ubuntu/)
```

Comment:
Maybe step 1 should be to clone the adept repo? Or maybe the dockerfile should clone the adept repo and all they need is the dockerfile?